neptune/internal/storage/datastream.py

```diff
@@ -64,11 +66,11 @@ def _get_chunk_size(self) -> int:
     def generate(self) -> Generator[FileChunk, Any, None]:
         chunk_size = self._get_chunk_size()
         last_offset = 0
-        last_change = os.stat(self._filename).st_mtime
+        last_change: Optional = os.stat(self._filename).st_mtime if self._filename else None
         while last_offset < self._total_size:
             chunk = self._fobj.read(chunk_size)
             if chunk:
-                if last_change < os.stat(self._filename).st_mtime:
+                if last_change and last_change < os.stat(self._filename).st_mtime:
```

Comment:
We made `last_change` an optional so it could be `None` and `if variable` ensures that it has a value. `None < <integer>` probably will throw a ValueError